src/docparser.cpp

```diff
@@ -1800,7 +1800,8 @@ static int internalValidatingParseDoc(DocNode *parent,DocNodeList &children,
 
   if (doc.isEmpty()) return retval;
 
-  doctokenizerYYinit(doc.data(),g_fileName,g_markdownSupport);
+  doctokenizerYYinit(doc.data(),g_fileName,g_markdownSupport,
+                     Config_getBool(QUOT_SUPPORT));
```

Comment:
Alternatively I can handle ``quotSupport`` like ``markdownSupport`` all the way through, i.e. create ``g_quotSupport`` and so on. However, that would involve various further changes throughout the code base. So far I did it the easy way to await your recommendation.